tasks/agent.py

```diff
@@ -152,16 +152,24 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
     }
     ctx.run(cmd.format(**args), env=env)
 
-    # Render the configuration file template
-    #
-    # We need to remove cross compiling bits if any because go generate must
-    # build and execute in the native platform
+    # Remove cross-compiling bits to render config
     env.update({
         "GOOS": "",
         "GOARCH": "",
     })
-    cmd = "go generate {}/cmd/agent"
-    ctx.run(cmd.format(REPO_PATH), env=env)
+
+    # Render the Agent configuration file template
+    if with_both_python:
```

Comment:
A nitpick here but could we use the same as before so that we don't duplicate args entirely ?

```python
cmd = "go run {go_file} {build_type} {template_path} {output_file}"

args = {
    "build_type": "agent-py2py3" if with_both_python else "agent-py3",
    "go_file": "./pkg/config/render_config.go",
    "output_file": "./cmd/agent/dist/datadog.yaml",
    "template_file": "./pkg/config/config_template.yaml",
}

ctx.run(cmd.format(**args), env=env)
```